app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>StaticPages#contact</h1>
```

Comment:
I18n cho toàn bộ các text ở file html nhé

Reply:
ok chị